### PR TITLE
Add Project description, SCM URL, Developer information to pom.xml

### PR DESCRIPTION
**Pull request by anwar6953**
Add Project description, SCM URL, Developer information to pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,6 +21,10 @@
   <groupId>co.cask.hydrator</groupId>
   <artifactId>hydrator-plugins</artifactId>
   <version>1.2.0</version>
+  <name>Hydrator Plugin Collection</name>
+  <description>A collection of plugins for Cask Hydrator.</description>
+  <url>http://docs.cdap.io/cdap/current/en/included-applications/index.html</url>
```
Comment (albertshau) [inline pom.xml]: http://docs.cdap.io/cdap/current/en/cdap-apps/hydrator/index.html

Comment (albertshau) [inline pom.xml]: actually, shouldn't this be https://github.com/caskdata/hydrator-plugins?

Comment (anwar6953) [inline pom.xml]: I just moved the url line from a few lines below.
Want me to change it to `https://github.com/caskdata/hydrator-plugins`?

Comment (albertshau) [inline pom.xml]: yeah think that makes more sense

**Comment by albertshau** [comment]
one comment then lgtm. fix then merge

**Pull request merged**